docs/src/gui/filter-programs.adoc

```diff
@@ -66,8 +66,8 @@ This feature should be used by any filter that runs for a long time.
 
 Here is a very basic example of the filtering mechanics:
 When run through a Linucnc screen that offers program filtering,
-it will produce and write a line of G-code every 100th of a second to standard output.
-It also sends a progress message out to the Unix standard error stream.
+it will produce and write a line of G-code every 100^th^ of a second to standard output.
```

Comment:
Why do you want to raise 'th' here?

Reply:
That is where it should always be, like 1<sup>st</sup> or 2<sup>nd</sup>.